src/common/utils/makeChartShareQuote.ts

```diff
@@ -33,5 +34,20 @@ export default function makeChartShareQuote(
       1,
     )} daily new cases per 100k population, according to @CovidActNow. See the chart: `;
   }
-  return `I'm keeping track of ${displayName}'s COVID data and risk level with @CovidActNow. What does your community look like?`;
+  return `I'm keeping track of ${displayName}'s vaccination progress and COVID risk level data with @CovidActNow. What does your community look like?`;
+}
+
+export function makeAddedChartShareQuote(
```

Comment:
nit: Not 100% sure why `Added` is in the function name here?  Is this because we recently added these charts or something?  I'd be a fan of trying to make names more future-proof so they make sense to somebody looking at this code in 6 months or whatever. 😁 

If there's a name collision we don't want to resolve right now, I'd at least add a TODO comment indicating the intended cleanup we should do later.

Reply:
lol yeah i didnt know what to call things related to the newly added charts